specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -626,8 +626,9 @@
           {
             "name": "opinionMining",
             "in": "query",
-            "description": "(Optional) if set to true, response will contain input and document level statistics including aspect-based sentiment analysis results.",
-            "type": "boolean"
+            "description": "(Optional) if set to true, response will contain input and document level statistics including (aspect-based) sentiment analysis results.",
+            "type": "boolean",
+            "default": true
```

Comment:
wouldn't this be a breaking change in the REST API?
we can have it be `true` by default in the SDKs without introducing breaking changes

Reply:
Yes, Assaf also mentioned this. I am turning this back to FASLE by default.